metron-interface/metron-config/src/app/login/login.component.html

```diff
@@ -0,0 +1,31 @@
+<!--
+  Licensed to the Apache Software
+	Foundation (ASF) under one or more contributor license agreements. See the
+	NOTICE file distributed with this work for additional information regarding
+	copyright ownership. The ASF licenses this file to You under the Apache License,
+	Version 2.0 (the "License"); you may not use this file except in compliance
+	with the License. You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software distributed
+	under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES
+	OR CONDITIONS OF ANY KIND, either express or implied. See the License for
+  the specific language governing permissions and limitations under the License.
+  -->
+<div class="fill login-pane" style="">
+  <div class="login-card">
```

Comment:
Forms are a slight overhead in a single page app. We need to interrupt the page submit and fire an ajax call. But I agree it would be nice to handle 'enter' key. A code change for this is submitted on this PR.